layers/+lang/perl5/packages.el

```diff
@@ -0,0 +1,117 @@
+          (if (eq (char-after (match-beginning 2)) 37)
+              'cperl-hash-face 'cperl-array-face) t)
+         ("\\(\\([$@]+\\)[a-zA-Z_:][a-zA-Z0-9_:]*\\)[ \t]*\\([[{]\\)" 1
+          (if (= (- (match-end 2) (match-beginning 2)) 1)
+              (if (eq (char-after (match-beginning 3)) 123)
+                  'cperl-hash-face 'cperl-array-face)
+            font-lock-variable-name-face) t)
+         ("\\([]}\\\\%@>*&]\\|\\$[a-zA-Z0-9_:]*\\)[ \t]*{[ \t]*\\(-?[a-zA-Z0-9_:]+\\)[ \t]*}"
+          (2 font-lock-string-face t)
+          ("\\=[ \t]*{[ \t]*\\(-?[a-zA-Z0-9_:]+\\)[ \t]*}" nil nil
+           (1 font-lock-string-face t)))
+         ("[[ \t{,(]\\(-?[a-zA-Z0-9_:]+\\)[ \t]*=>" 1 font-lock-string-face t)))
+
+      (font-lock-add-keywords
+       'cperl-mode
+       '(("\\(\\([@%]\\|\\$#\\)[a-zA-Z_:][a-zA-Z0-9_:]*\\)" 1
+          (if (nth 4 (syntax-ppss))
+              'font-lock-comment-face
+            (if (eq (char-after (match-beginning 2)) ?%)
+                'cperl-hash-face
+              'cperl-array-face)) t)
+         ("\\(\\([$@]+\\)[a-zA-Z_:][a-zA-Z0-9_:]*\\)[ \t]*\\([[{]\\)" 1
+          (if (nth 4 (syntax-ppss))
+              'font-lock-comment-face
+            (if (= (- (match-end 2) (match-beginning 2)) 1)
+                (if (eq (char-after (match-beginning 3)) ?{)
+                    'cperl-hash-face
+                  'cperl-array-face)
+              font-lock-variable-name-face)) t)
+         ("\\([]}\\\\%@>*&]\\|\\$[a-zA-Z0-9_:]*\\)[ \t]*{[ \t]*\\(-?[a-zA-Z0-9_:]+\\)[ \t]*}"
+          (2 (if (nth 4 (syntax-ppss))
+                 'font-lock-comment-face
+               'font-lock-string-face) t)
+          ("\\=[ \t]*{[ \t]*\\(-?[a-zA-Z0-9_:]+\\)[ \t]*}" nil nil
+           (1 (if (nth 4 (syntax-ppss))
+                  'font-lock-comment-face
+                'font-lock-string-face) t)))
+         ("[[ \t{,(]\\(-?[a-zA-Z0-9_:]+\\)[ \t]*=>" 1
+          (if (nth 4 (syntax-ppss))
+              'font-lock-comment-face
+            'font-lock-string-face) t)))
+
+      ;; Use less horrible colors for cperl arrays and hashes
+      (set-face-attribute 'cperl-array-face nil :foreground  "#DD7D0A"    :background  'unspecified :weight 'unspecified)
+      (set-face-attribute 'cperl-hash-face nil  :foreground  "OrangeRed3" :background  'unspecified :weight 'unspecified)
+      (setq cperl-highlight-variables-indiscriminately t)
+
+      ;; default settings
+      (setq cperl-indent-level 4) ;; 4 spaces is the standard indentation
+      (setq cperl-close-paren-offset -4) ;; indent the closing paren back four spaces
+      (setq cperl-continued-statement-offset 4) ;; if a statement continues indent it to four spaces
+      (setq cperl-indent-parens-as-block t) ;; parentheses are indented with the block and not with scope
+
+      ;; tab key will indent all marked code when pressed
+      (add-hook 'cperl-mode-hook
+                (lambda () (local-set-key (kbd "<tab>") 'indent-for-tab-command)))
```

Comment:
I added this back in Spacemacs 0.105.11 because hitting tab key would not indent the code properly and default to some cperl function. However I removed the above code and everything seems to work fine on the latest version. I will go ahead and remove it.